js/whitebit.js

```diff
@@ -280,6 +297,7 @@ module.exports = class whitebit extends Exchange {
             const quote = this.safeCurrencyCode (quoteId);
             const symbol = base + '/' + quote;
             const active = this.safeValue (market, 'tradesEnabled');
+            const isMargin = this.inArray (id, marginMarkets);
```

Comment:
@Dan-krm I've already fixed it now, but for future reference, we should be using `this.inArray ();` every time we want to see if an element exists inside an array, `indexOf` breaks the other languages